climada/hazard/centroids/test/test_centr.py

```diff
@@ -130,11 +130,33 @@ def test_from_csv(self):
         # Read centroids using from_csv method
         centroids = Centroids.from_csv(tmpfile)
 
-        # test attributes
+        # Test attributes
         np.testing.assert_array_equal(centroids.lat, lat)
         np.testing.assert_array_equal(centroids.lon, lon)
         self.assertEqual(centroids.crs, 'epsg:3857')
 
+        # Delete file
+        Path(tmpfile).unlink()  
+    
+    def test_write_read_csv(self):
+        """Write and read a Centroids CSV file correctly."""
+        # Create Centroids with latitude and longitude arrays
+        tmpfile = Path('test_write_csv.csv')
+        lat = np.array([10.0, 20.0, 30.0])
+        lon = np.array([-10.0, -20.0, -30.0])
+        centroids_out = Centroids(lat=lat, lon=lon)
+
+        # Write CSV file from Centroids using write_csv
+        centroids_out.write_csv(tmpfile)
+
+        # Read CSV file using read_csv
+        centroids_in = Centroids.from_csv(tmpfile)
+
+        # Test attributes
+        np.testing.assert_array_equal(centroids_in.lat, centroids_out.lat)
+        np.testing.assert_array_equal(centroids_in.lon, centroids_out.lon)
+        self.assertEqual(centroids_in.crs, centroids_out.crs)
```

Comment:
This is a bit dangerous, as the `crs` can be written in different manners, and the strings might not be identical before/after loading. Better to use the `util.coordinates.equal_crs` method.

Reply:
I can of course change it but this is checking the crs of the centroids gdf, not the strings in the file, so it shouldn't be an issue no?

Reply:
In theory, I agree, but I remember having some trouble with the equality of `crs`. So I would go for the more cautious approach here.